sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -641,8 +641,8 @@ class SQLContext private[sql](val sparkSession: SparkSession)
   }
 
   /**
-   * Executes a SQL query using Spark, returning the result as a `DataFrame`. The dialect that is
-   * used for SQL parsing can be configured with 'spark.sql.dialect'.
+   * Executes a SQL query using Spark, returning the result as a `DataFrame`.
+   * The dialect that is used for SQL parsing can be configured with 'spark.sql.dialect'.
```

Comment:
Let's remove it.

Reply:
the doc here should be the same as `SparkSession.sql`

Reply:
Oh, that was a mistake. Fixed.